plaso/cli/psteal_tool.py

```diff
@@ -316,7 +321,17 @@ def ExtractEventsFromSources(self):
     self._SetExtractionPreferredTimeZone(extraction_engine.knowledge_base)
 
     filter_find_specs = None
-    if configuration.filter_file:
+    if configuration.artifacts_filter_file:
```

Comment:
repeat of earlier comment about helper method

Reply:
Moved this to the tools.py so that it can be shared by image_export, log2timeline and psteal.